skills/create-auth/reference/create-auth.md

```
@@ -0,0 +1,327 @@
+
+---
+
+## Environment variables
+
+Never hardcode secrets.
+
+Prefer automatic environment variable loading.
+
+Aura Auth automatically looks for
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)